trunk/wc_yotpo.php

```diff
@@ -216,9 +216,12 @@ function generate_star_ratings_widget_code() {
 	$use_v3_widgets = $yotpo_settings['use_v3_widgets'];
 	$reviews_widget_id = $yotpo_settings['reviews_widget_id'];
 	$star_ratings_widget_id = $yotpo_settings['star_ratings_widget_id'];
-	return $use_v3_widgets && $reviews_widget_id && $star_ratings_widget_id
-		? generate_v3_star_ratings_widget_code($star_ratings_widget_id)
-		: generate_v2_star_ratings_widget_code();
+	if ($use_v3_widgets && $star_ratings_widget_id && $reviews_widget_id) {
+		return generate_v3_star_ratings_widget_code($star_ratings_widget_id);
+	}
+	if (!$use_v3_widgets) {
```

Comment:
Are there any further cases where nothing will be returned over here?
If not, this if statement is redundant.